command-line.dylan

```diff
@@ -56,6 +56,11 @@ define function parse-args
                   variable: "FILE",
                   help: "File in which to store the report."));
 
+  add-option(parser,
+             make(<repeated-parameter-option>,
+                  names: #("load"),
+                  help: "Load the given shared library before searching for test suites. May be repeated."));
+
```

Comment:
I think this should be specific about whether the library name is enough or if a full pathname is needed.